src/main/java/seedu/address/logic/parser/recipe/AddRecipeCommandParser.java

```diff
@@ -69,6 +69,10 @@ public AddRecipeCommand parse(String args) throws ParseException, IOException, U
         String instructionString = argMultimap.getValue(PREFIX_INSTRUCTION).get();
         ArrayList<Instruction> instructions = InstructionParser.parse(instructionString);
 
+        if (instructions.size() == 0) {
```

Comment:
isnt this exact if statement already checked in instructionparser? is it redundant? maybe an assert would be better?